cmd/poseidon/main.go

```diff
@@ -156,17 +158,36 @@ func runServer(server *http.Server, cancel context.CancelFunc) {
 	defer cancel()
 	defer shutdownSentry() // shutdownSentry must be executed in the main goroutine.
 
-	log.WithField("address", server.Addr).Info("Starting server")
+	// Creating http Listener
+	var httpListener net.Listener
 	var err error
+	if config.Config.Server.SystemdSocketActivation {
+		var listeners []net.Listener
+		listeners, err = activation.Listeners()
+		if len(listeners) > 0 {
+			httpListener = listeners[0]
+		}
```

Comment:
At first, I was somewhat surprised by using `listeners[0]`. However, I understand that a systemd socket may contain multiple `ListenStream=` directives and hence multiple sockets. However, in Poseidon, we don't support multiple `ListenStream=` entries and will discard all others except the first one, right? Should we either document that or, listen on all provided sockets (which could be used for simultaneous IPv4 and IPv6 support on the same interface)?